tests/test_wave_01.py

```diff
@@ -49,24 +52,33 @@ def test_get_task(client, one_task):
             "is_complete": False
         }
     }
+    new_task = Task.query.get(1)
+    assert new_task
+    assert new_task.title == "Go on my daily walk 🏞"
+    assert new_task.description == "Notice something new every day"
+    assert new_task.completed_at == None
 
 
-@pytest.mark.skip(reason="No way to test this feature yet")
 def test_get_task_not_found(client):
     # Act
     response = client.get("/tasks/1")
     response_body = response.get_json()
 
     # Assert
     assert response.status_code == 404
+    assert response_body == {"message":"Task 1 not found"}
+    assert Task.query.all() == []
+
 
-    raise Exception("Complete test with assertion about response body")
-    # *****************************************************************
-    # **Complete test with assertion about response body***************
-    # *****************************************************************
+def test_get_task_invalid_id(client):
+    response = client.get("/tasks/cat")
+    response_body = response.get_json()
+
+    assert response.status_code == 400
+    assert response_body == {"message": "Task cat invalid"}
+    assert Task.query.all() == []
```

Comment:
Great added test idea. Exactly the type of thing that you should test!

Reply:
Same with the other tests of invalid input in this file, but I won't point out every one, as not to repeat myself. Yes, I even do it for the positive comments.